src/components/UserContributionForm.vue

```diff
@@ -0,0 +1,130 @@
+<script setup>
+import { ref, computed } from "vue";
+import { defineProps } from "vue";
+import { recordUserContribution } from "../stores/api.js";
+
+const data = defineProps({
+  description: String,
+  inputOptions: Object,
+  missingFieldName: String,
+  toolName: String,
+  taskId: Number,
+  isError: Boolean,
+  getNextTask: Function,
+});
+
+const inputOptionsArray = computed(() => {
+  return data?.inputOptions
+    ? Object.entries(data.inputOptions).map(([key, value]) => ({ key, value }))
+    : [];
+});
+
+const missingFieldValue = ref(null);
+const errorMessage = ref("");
+
+const submit = async () => {
+  const fieldValue = missingFieldValue.value;
+  const contributionRecord = {
+    value: Array.isArray(fieldValue) ? fieldValue.join() : fieldValue,
+    field: data.missingFieldName,
+    tool: data.toolName,
+  };
+
+  try {
+    const res = await recordUserContribution(data.taskId, contributionRecord);
+    alert(res)
+  } catch (error) {
+    console.log(error);
+  } finally {
+    missingFieldValue.value = null;
+  }
+};
+
+async function postUserContribution() {
+  if (missingFieldValue.value.length > 0) {
+    const contributionRecord = {
+      value: Array.isArray(missingFieldValue.value)
+        ? missingFieldValue.value.join()
+        : missingFieldValue.value,
+      field: data.missingFieldName,
+      tool: data.toolName,
+    };
+
+    try {
+      await recordUserContribution(data.taskId, contributionRecord);
+    } catch (error) {
+      console.log(error);
+    } finally {
+      dialog.value = false;
+      missingFieldValue.value = null;
+    }
+  }
+}
+
+function skipToNext() {
+  missingFieldValue.value = null;
+  data.getNextTask();
+}
+</script>
+
+<template>
+  <v-container>
+    <v-row>
+      <v-col cols="12">
+        <p class="text-h6" >{{ description }}</p>
```

Comment:
This paragraph text has the styles of a header. The implication of this on the UI currently is that even though it's the description of the edited tool, it appears to be the most obvious text on the screen given it size. 

Can we use a more appropriate style (preferably smaller and less bold) and maybe see what it'd look like if we place it below the input field instead of at the top.

Reply:
Ohh just saw the mock up you added to the PR. If the intention is to make the description the title of the field, then we could use the v-card-title instead of the p tag.